modules/osdk-cli-reference-run.adoc

```diff
@@ -7,54 +7,36 @@
 
 The `operator-sdk run` command provides options that can launch the Operator in various environments.
 
-.`run` arguments
+== packagemanifests
+
+`run packagemanifests` subcommand deploys an Operator's package manifests with OLM. The command argument must be set to a valid package manifest root directory, for example, ``<project-root>/packagemanifests`.
+
+.`packagemanifests` arguments
 [options="header",cols="1,3"]
 |===
 |Arguments |Description
 
+|`--include` (string)
+|The file path to Kubernetes resource manifests, such as `Role` objects and `Subscription` objects. These supplement or override defaults generated by `run` or `cleanup`.
```

Comment:
The formatting for "`Role` objects and `Subscription` objects" is fine as you have it for an object reference.  Please just make sure that the formatting in this file and the `modules/osdk-cli-reference-cleanup.adoc` file are identical.